force-app/main/default/classes/SOQL.cls

```diff
@@ -213,14 +257,15 @@ public with sharing class SOQL {
      */
     public static Set<Id> getCaseIdsByAccount(Id accountId) {
         // Create a set to hold the Case Ids
-
+        Set<Id> caseIds = new Set<Id>();
         // Query the Cases
+        List<Case> cases = [SELECT Id FROM Case WHERE AccountId = :accountId];
         // Loop through each Case and add the Id to the set
-        //for () {
-        //    
-        //}
+        for (Case c : cases) {
+            caseIds.add(c.Id);
+        }
 
```

Comment:
If you are looking for a Set of Ids, you can also query right into a Map and then use the map keySet() method. That prevents you from needing to loop manually and add to the set.

Reply:
Yes, that is much more elegant. However, I was following the inline comments/instructions so didn't take that approach, but definitely better. Making that update.